hibernate-reactive-core/src/test/java/org/hibernate/reactive/FetchModeSelectEagerTest.java

```diff
@@ -0,0 +1,250 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive;
+
+import io.vertx.ext.unit.TestContext;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+import org.hibernate.cfg.Configuration;
+import org.junit.Test;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class FetchModeSelectEagerTest extends BaseReactiveTest {
```

Comment:
One more minor thing, @gbadner, I always actually intended for `EagerTest` to be changed to test select fetching for the parent as well. So I don't think we need `FetchModeSelectEagerTest`, we just need to change the fetch mode of `Element.node`, uncommenting the commented line.

Reply:
@gavinking , OK, I'll change `EagerTest` and remove `FetchModeSelectEagerTest`.

Reply:
Actually, I'll change `FetchModeSelectEagerTest` to `FetchModeSubselectEagerTest` and look into why this is failing for `FetchMode.SUBSELECT`.

Reply:
Awesome, thanks.